docs/modular-resources/advanced/create-subtype.md

```diff
@@ -4,76 +4,83 @@ linkTitle: "New API Subtype"
 weight: 30
 type: "docs"
 tags: ["rdk", "extending viam", "modular resources", "API"]
-description: "Define a new resource subtype using an existing resource API."
+description: "Define a new resource subtype and API for a resource that does not fit into existing component or service subtypes."
 no_list: true
 aliases:
   - "/extend/modular-resources/create/create-subtype/"
 ---
 
-{{% alert title="Tip" color="tip" %}}
+You can define a new resource _{{< glossary_tooltip term_id="subtype" text="subtype" >}}_ (and an API for that subtype) if:
 
-[Modular resources](/modular-resources/) provided by custom {{< glossary_tooltip term_id="module" text="modules" >}} are the preferred method of creating custom resource implementations.
-
-This option is provided for advanced users who are unable to use the modular resource system with the robot deployments.
-
-{{% /alert %}}
-
-You need to define a new resource _subtype_ (and an API for that subtype) if:
-
-- You have a {{% glossary_tooltip term_id="resource" text="resource" %}} that does not fit into any of the existing [component](/components/) or [service](/services/) subtypes.
+- You have a {{% glossary_tooltip term_id="resource" text="resource" %}} that does not fit into any of the existing {{< glossary_tooltip term_id="component" text="component" >}} or {{< glossary_tooltip term_id="service" text="service" >}} subtypes.
 - You have a resource that could fit into an existing subtype, but you want to define an API with different methods and messages than the ones in the existing [APIs](/program/apis/) for that subtype.
   For example, you have a [sensor](/components/sensor/) and you want to define a `Calibrate` method.
```

Comment:
Maybe we should actually remove this--it might be a good example for what you should do with DoCommand. Dunno what happened with the de-emphasis ticket but this might tie into that.